src/lib/containers/IconSvg.tsx

```diff
@@ -374,7 +377,9 @@ const getIcon = (options: IconSvgOptions) => {
     case 'discussion':
       return <QuestionAnswerTwoTone style={muiSvgStyle} />
     case 'dataset':
-      return <GridOnTwoTone style={muiSvgStyle} />
```

Comment:
Is the GridOnTwoTone being used elsewhere or can we remove it from import?

Reply:
Good call, probably not used.  I'll check